Orange/widgets/data/owpivot.py

```diff
@@ -821,6 +829,11 @@ def commit(self):
         self.Outputs.pivot_table.send(self.pivot.pivot_table)
         self.Outputs.filtered_data.send(self.get_filtered_data())
 
+        output = self.pivot.pivot_table
```

Comment:
I'd rather consider `filtered data` here.

Reply:
Even though `pivot table` is the default output of the widget and if a user wants filtered data on the output they have to change connection afterwards?

Reply:
Yes.